src/LineageTree/lineageTree.py

```diff
@@ -481,7 +480,8 @@ def to_treex(self, sampling=1, start=0, finish=10000, many=True):
         times_to_consider = sorted([
             t for t, n in self.time_nodes.items() if 0 < len(n)
         ])
-        times_to_consider = times_to_consider[start:finish:sampling]
+        times_to_consider = times_to_consider[::sampling]
```

Comment:
Because when a user  selects a start he is thinking about the starting timepoints and not the resampled ones. (that is how i think about it when selecting a start time)

Reply:
I agree with what you say.
But, would you agree that the two pieces of code do exactly the same things for the same values of `start`, `finish` and `sampling`?

To convince you, maybe try to do that:
```python
import numpy as np
start = 2
finish = 20
times_to_consider = np.arange(25)

# your 4-line version
start //= sampling
finish //= sampling
times_to_consider_2 = times_to_consider[::sampling]
times_to_consider_2 = times_to_consider_2[start:finish]

start = 2
finish = 20
# my one line equivalent, imo more readable version
times_to_consider_3 = times_to_consider[start:finish:sampling]

# Making sure that they are the same
(times_to_consider_2 == times_to_consider_3).all()
```
So the user would give the same input and get the same output right?

Reply:
for start < sampling 
![Screenshot from 2023-08-23 17-53-38](https://github.com/leoguignard/LineageTree/assets/112393764/386b17b3-f62e-4cbe-b664-98c80e953b08)

Reply:
When I'm applying the to_treex I expect to always get the starting point I want, because if not I may miss a division or more